src/components/FileDetail/FileDetail.js

```diff
@@ -173,7 +173,7 @@ const FileDetail = ({ fileNode, history, match }) => {
                     </Segment.Group>
 
                   </Grid.Column>
-                  <Grid.Column mobile={8} tablet={8} computer={3}>
+                  <Grid.Column mobile={8} tablet={4} computer={3}>
```

Comment:
There's probably some more work to do with making the actions more responsive in the future. Maybe inline them on tablet devices.